### PR TITLE
[OBMIN-273] created avatar

### PR DESCRIPTION
**Pull request by Kyslytsia**
created avatar 
created image

### CODE REVIEW

File: src/components/avatar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+import * as Icon from '../icon';
+
```
Comment (Wolshebnik) [inline src/components/avatar/index.tsx:4]: Delete space

File: src/components/image/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+
+import * as Icon from '../icon';
+
```
Comment (Wolshebnik) [inline src/components/image/index.tsx:4]: the same

File: src/components/image/types.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export interface ImageProps {
+  width?: number;
+  height?: number;
+  source: string | undefined;
```
Comment (Wolshebnik) [inline src/components/image/types.ts:4]: If you received undefined
You should write 
source?: string;

File: src/components/avatar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    if (!source?.includes(string) && source) {
+      setImage(`${string}${source}`);
+    }
+
+    if (source?.includes(string) && source) {
+      setImage(source);
+    }
+
+    if (!source) setImage(undefined);
+  }, [source]);
```
Comment (Wolshebnik) [inline src/components/avatar/index.tsx:23]: I think we need to remove useEffect from this file and past in Avatar.stories.ts

**Review by Wolshebnik — APPROVED** [review]

**Review by AkintsevaElizaveta — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

**Pull request merged**